### PR TITLE
[SPARK-12814][Document]Add deploy instructions for Python in flume integration doc

### PR DESCRIPTION
**Pull request by zsxwing**
This PR added instructions to get flume assembly jar for Python users in the flume integration page like Kafka doc.

### CODE REVIEW

**Comment by zsxwing** [comment]
CC @tdas

**Comment by zsxwing** [comment]
This PR mostly copied the content from Kafka integration page and changed `kafka` to `flume`.

File: docs/streaming-flume-integration.md [hunk 1]
```diff
@@ -157,7 +166,7 @@ configuring Flume agents.
 
 	Note that each input DStream can be configured to receive data from multiple sinks.
 
-3. **Deploying:** Package `spark-streaming-flume_{{site.SCALA_BINARY_VERSION}}` and its dependencies (except `spark-core_{{site.SCALA_BINARY_VERSION}}` and `spark-streaming_{{site.SCALA_BINARY_VERSION}}` which are provided by `spark-submit`) into the application JAR. Then use `spark-submit` to launch your application (see [Deploying section](streaming-programming-guide.html#deploying-applications) in the main programming guide).
+3. **Deploying:** This is same as the first approach, for Scala, Java and Python.
```
Comment (tdas) [inline docs/streaming-flume-integration.md]: `approach.`

File: docs/streaming-kafka-integration.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Next, we discuss how to use this approach in your streaming application.
 	    ./bin/spark-submit --packages org.apache.spark:spark-streaming-kafka_{{site.SCALA_BINARY_VERSION}}:{{site.SPARK_VERSION_SHORT}} ...
 
 	Alternatively, you can also download the JAR of the Maven artifact `spark-streaming-kafka-assembly` from the 
-	[Maven repository](http://search.maven.org/#search|ga|1|a%3A%22spark-streaming-kafka-assembly_2.10%22%20AND%20v%3A%22{{site.SPARK_VERSION_SHORT}}%22) and add it to `spark-submit` with `--jars`.
+	[Maven repository](http://search.maven.org/#search|ga|1|a%3A%22spark-streaming-kafka-assembly_{{site.SCALA_BINARY_VERSION}}%22%20AND%20v%3A%22{{site.SPARK_VERSION_SHORT}}%22) and add it to `spark-submit` with `--jars`.
```
Comment (tdas) [inline docs/streaming-kafka-integration.md]: Does this problem exist in the other guides? Kinesis?

**Comment by tdas** [comment]
LGTM. Just couple of nits. Thanks @zsxwing

File: docs/streaming-flume-integration.md (hunk 1 above)
Comment (zsxwing) [inline docs/streaming-flume-integration.md]: Did you mean `,` -> `.`?

File: docs/streaming-kafka-integration.md (hunk 2 above)
Comment (zsxwing) [inline docs/streaming-kafka-integration.md]: No. Kinesis doc doesn't contain a maven link.

Comment (tdas) [inline docs/streaming-kafka-integration.md]: May be it should contains similar docs. That'sa different PR anyways.

File: docs/streaming-flume-integration.md (hunk 1 above)
Comment (tdas) [inline docs/streaming-flume-integration.md]: yes, and remove Scala JAva Python

Comment (zsxwing) [inline docs/streaming-flume-integration.md]: Done.

File: docs/streaming-kafka-integration.md (hunk 2 above)
Comment (zsxwing) [inline docs/streaming-kafka-integration.md]: > May be it should contains similar docs. That'sa different PR anyways.

Okey, I will open another PR for Kinesis.

**Comment by tdas** [comment]
LGTM. Merging.

File: docs/streaming-kafka-integration.md (hunk 2 above)
Comment (zsxwing) [inline docs/streaming-kafka-integration.md]: See #10822 for kinesis doc changes.

**Pull request closed without merging**